planner/core/prepare_test.go

```diff
@@ -28,7 +28,9 @@ import (
 	"github.com/pingcap/tidb/infoschema"
 	"github.com/pingcap/tidb/kv"
 	"github.com/pingcap/tidb/metrics"
+	"github.com/pingcap/tidb/parser"
```

Comment:
all the change from this file are moved from physical_plan_test.go

Reply:
`prepare_test.go` has been migrated, please rebase on the master.